### ISSUE TITLE
[Bug]: Alma Linux as a client cannot ping server

### ISSUE THREAD

**Issue by xoraingroup** [gravitl/netmaker#2300]
### What happened?

1. Try to spin test server with 

```
sudo wget -qO /root/nm-quick.sh https://raw.githubusercontent.com/gravitl/netmaker/master/scripts/nm-quick.sh && sudo chmod +x /root/nm-quick.sh && sudo /root/nm-quick.sh
```
2. Create any ipv4/ipv6 network
3. Get an Alma linux vm and install netclient using 

```
curl -sL 'https://rpm.netmaker.org/gpg.key' | sudo tee /tmp/gpg.key
curl -sL 'https://rpm.netmaker.org/netclient-repo' | sudo tee /etc/yum.repos.d/netclient.repo
sudo rpm --import /tmp/gpg.key
sudo dnf check-update
sudo dnf install netclient
```

4. Register netclient to server using 

```
netclient register -t token
```

5.  Alma linux vm successfully connects.

Try to ping with assigned ipv4 or ipv6 from alma linux vm to server and vice versa.

Expected Result:
Ping should work.

Actual Result:
Ping does not work.

It works even with Rocky Linux with above steps but does not work with Alma linux.

Tested with brand new vm from hetzner with no firewall, still didn't work. Tested with other providers same result. Alma linux does not work, but Rocky Linux does work.

### Version

v0.19.0

### What OS are you using?

Linux

### Contributing guidelines

- [X] Yes, I did.

**Comment by darkerdead** [comment]
Huh, I have been trying to ssh to my home server from work all day and it keeps timing out. I thought it's my installation so I tried re-installing netmaker on my vps. Heck I even rebuilt the entire vps because it wasn't working. Didn't even consider it might a bug or some issue from netmaker. I guess, I can get an update here.

**Comment by xoraingroup** [comment]
@darkerdead Not really, it just works out of the box as soon i register or if i use ```netclient join -t token```.  It worked on debian 11, ubuntu 22.04 and even rocky linux new vps without any firewalls etc. But with alma linux it shows that its connected and  ```ip a l ``` shown netmaker interface as down. Tried every piece of information in documentation to make it work but it didn't, thats why created a bug report here.

**Comment by xoraingroup** [comment]
Just to add an update here.

It works with Alma linux 9 but not with Alma linux 8.

**Comment by mattkasun** [comment]
could you post the output of journalctl -u netclient

**Comment by mattkasun** [comment]
does Alma linux 8 have wireguard kernel module?

**Comment by xoraingroup** [comment]
Yes it had. Right now I am traveling but I will put the info here as soon I am back. 

Thank you

**Comment by mattkasun** [comment]
What is output of `wg show`
